### ISSUE TITLE
Code generator works wrong

### ISSUE THREAD

**Issue by mykola-dev** [yatatsu/AutoBundle#2]
Here is generated class:

```
public final class TariffAcceptingWithOffersActivityAutoBundle {
  public static com.matrixx.android.activity.IntentBuilder createIntentBuilder(boolean fromSurvey) {
    return new com.matrixx.android.activity.IntentBuilder(fromSurvey);
  }

  public static void bind(TariffAcceptingWithOffersActivity target, Intent intent) {
    if (intent.getExtras() != null) {
      bind(target, intent.getExtras());
    }
  }

  public static void bind(TariffAcceptingWithOffersActivity target, Bundle source) {
    if (source.containsKey("fromSurvey")) {
      target.fromSurvey = (boolean) source.getBoolean("fromSurvey");
    } else {
      throw new IllegalStateException("fromSurvey is required, but not found in the bundle.");
    }
  }

  public static void pack(TariffAcceptingWithOffersActivity source, Bundle args) {
    args.putBoolean("fromSurvey", source.fromSurvey);
  }

  public static final class IntentBuilder {
    final Bundle args;

    public IntentBuilder(boolean fromSurvey) {
      this.args = new Bundle();
      this.args.putBoolean("fromSurvey", fromSurvey);
    }

    public Intent build(Context context) {
      Intent intent = new Intent(context, TariffAcceptingWithOffersActivity.class);
      intent.putExtras(args);
      return intent;
    }

    public Intent build(Intent intent) {
      intent.putExtras(args);
      return intent;
    }
  }
}
```

It fails on resolving IntentBuilder
http://i.imgur.com/WJ7c5uq.png

**Comment by yatatsu** [comment]
Thanks for your feedback. It seems to be wrong certainly. 
Could you show more detail? I'd like to see original and generated one.

**Comment by mykola-dev** [comment]
My activity is in com.matrixx.android.activity package
activity has 1 @AutoBundle field
i make binding in onCreate.
Looks like generator only works ok when activity placed in the project root package

**Comment by yatatsu** [comment]
I see. It may be occurred in specific package placement.
I'll try to reproduce it. Thanks.

**Comment by yatatsu** [comment]
Caller method for builder pointed to wrong package actually. 
I Fixed it, so try in `1.0.2`.

```
compile 'com.github.yatatsu:autobundle:1.0.2'
apt 'com.github.yatatsu:autobundle-processor:1.0.2'
```

**Comment by mykola-dev** [comment]
thanks! it works now
